iroh/src/rpc_protocol/net.rs

```diff
@@ -37,6 +37,7 @@ pub enum Response {
     RemoteInfosIter(RpcResult<RemoteInfosIterResponse>),
     RemoteInfo(RpcResult<RemoteInfoResponse>),
     Watch(WatchResponse),
+    Unit(RpcResult<()>),
```

Comment:
what's this?

Reply:
Some rpc calls return RpcResult<()>. We have to have a way to get the result back.